### ISSUE TITLE
vue support autorun test-src-file

### ISSUE THREAD

**Issue by GFdevelop** [jest-community/vscode-jest#790]
### Environment

1. `vscode-jest version`: 4,2,1
2. `node -v`: 16.11.1
3. `npm -v` or `yarn --version`: 8.1.0
4. `npm ls jest` or `npm ls react-scripts` (if you haven’t ejected): 24.9.0
5. your vscode-jest settings if customized:
    - jest.jestCommandLine? `npm run test:unit --`
    - jest.autoRun? `{ "watch": false, "onSave": "test-src-file" }`
    - anything else that you think might be relevant?
6. Operating system: Arch Linux

### Prerequisite
- are you able to run jest test from the command line? yes
- how do you run your tests from the command line? (for example: `npm run test` or `node_modules/.bin/jest`) `npm tun test:unit`

### Steps to Reproduce

1) create a new vue project with `vue create foo` and select unit test with jest
2) change the Helloworld.vue and save it: the test isn't triggered

### Relevant Debug Info

On the bottom right if you change the filetype from "vue" to "javascript" there isn't the issue.

### Expected Behavior

After save the source file (ex: foo.vue) the extension should run the test (foo.spec.js)

### Actual Behavior

The test is running only after change the test file

--- 

The fastest (and the most fun) way to resolve the issue is to submit a pull-request yourself. If you are interested, feel free to check out the [contribution guide](https://github.com/jest-community/vscode-jest/CONTRIBUTING.md), we look forward to seeing your PR...

**Comment by connectdotz** [comment]
@GFdevelop thanks for testing the fix:

> Click the "Debug Tests" button give me...

The extension currently supports test-block level debug only, do you by any chance clicked the debug button on the file or folder level? (We do have an action item to remove the debug button from those unsupported entities, which just became possible in the new Test Explorer API) I assumed all vue test files will be just regular `ts(x)` or `js(x)` files, is that correct?

> I don't known how to show the coverage: give me the steps to test the coverage.

sure. you can toggle the coverage by running the command `Jest: Toggle Coverage` from the Command Palette.

**Comment by GFdevelop** [comment]
> do you by any chance clicked the debug button on the file or folder level? 

Yes, I get that error on the file or folder level

> I assumed all vue test files will be just regular `ts(x)` or `js(x)` files, is that correct?

Yes

> sure. you can toggle the coverage by running the command `Jest: Toggle Coverage` from the Command Palette.

I can see some lines highlighted, but I don't known if this is the expected result...what lines must be highlighted?

**Comment by connectdotz** [comment]
ok, great, are you able to debug the individual test then? If true then we can check-off debug feature for vue.

regrading coverage, the highlighted lines are either not-tested or partially tested, see more detail by expanding "How to read coverage scheme and customize it" in the [README](https://github.com/jest-community/vscode-jest#how-to-use-code-coverage)

**Comment by GFdevelop** [comment]
> ok, great, are you able to debug the individual test then? If true then we can check-off debug feature for vue.

I can run the single test, but debug the single test start the debugger and this message appear:
```
No debug config named "vscode-jest-tests.v2" or "vscode-jest-tests" found in launch.json, will use a default config.
If you encountered debugging problems, feel free to try the setup wizard below
```

> regrading coverage, the highlighted lines are either not-tested or partially tested, see more detail by expanding "How to read coverage scheme and customize it" in the [README](https://github.com/jest-community/vscode-jest#how-to-use-code-coverage)

Before I can see some red lines, now I can't see colors on lines and I don't understand what is wrong.

**Comment by connectdotz** [comment]
> I can run the single test, but debug the single test start the debugger and this message appear

that's ok, we should probably turn it off in the future release. As long as you can debug the test, it's cool...

> Before I can see some red lines, now I can't see colors on lines and I don't understand what is wrong.

First, make you did have the coverage toggled on as we discussed [here](https://github.com/jest-community/vscode-jest/issues/790#issuecomment-965573750). If it is 100% covered, you might also not seeing any color, but there should always be a summary line on the top of your source file.

**Comment by GFdevelop** [comment]
> First, make you did have the coverage toggled on as we discussed [here](https://github.com/jest-community/vscode-jest/issues/790#issuecomment-965573750). If it is 100% covered, you might also not seeing any color, but there should always be a summary line on the top of your source file. 

I have only one test file, the coverage must be <=5% and I don't have the summary line.

**Comment by connectdotz** [comment]
ok, this is very helpful 👍 , I found the issue, will patch it soon

**Comment by connectdotz** [comment]
ok, I fixed a few issues, please feel free to try the updated vsix file in #791 

Please note that you would only see coverage summary/info for the file you have tested, in this case only `HelloWorld.vue` will show test coverage summary. If you want to see jest generate coverage info for all files, including those without tests, you should add ["collectCoverageFrom"](https://jestjs.io/docs/configuration#collectcoveragefrom-array) to your jest config.

**Comment by connectdotz** [comment]
@GFdevelop I have checked in the PR and closed this issue. Feel free to reopen it or create a new issue if you encounter other related issues.
